pkgs/games/anki/default.nix

```
@@ -157,17 +154,23 @@ buildPythonApplication rec {
       cp -rv locale $out/share/
       cp -rv anki aqt web $pp/
 
-      wrapPythonPrograms
-
       # copy the manual into $doc
       cp -r ${manual}/share/doc/anki/html $doc/share/doc/anki
     '';
 
+    dontWrapQtApps = true;
+    makeWrapperArgs = [
+        ''--prefix PATH ':' "${lame}/bin:${mplayer}/bin"''
```

Should we use the same string type for both array elements?

```suggestion
        "--prefix PATH ':' ${lame}/bin:${mplayer}/bin"
```